src/Plugin/PatternLibrary/PatternLibraryFile.php

```diff
@@ -3,6 +3,7 @@
 namespace Drupal\patternkit\Plugin\PatternLibrary;
 
 use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\patternkit\Entity\PatternInterface;
 use Drupal\patternkit\Pattern;
```

Comment:
It seems as though this line `use Drupal\patternkit\Pattern;` was intended to be removed in favor of `use Drupal\patternkit\Entity\PatternInterface;` Is that correct?

Reply:
Fixed, thanks!